README.md

```diff
@@ -1,6 +1,102 @@
-# Percy Capybara
+# percy-capybara
+[![Gem Version](https://badge.fury.io/rb/percy-capybara.svg)](https://badge.fury.io/rb/percy-capybara)
+![Test](https://github.com/percy/percy-capybara/workflows/Test/badge.svg)
 
-[![CircleCI](https://circleci.com/gh/percy/percy-capybara.svg?style=svg)](https://circleci.com/gh/percy/percy-capybara)
-[![Gem Version](https://badge.fury.io/rb/percy-capybara.svg)](http://badge.fury.io/rb/percy-capybara)
+[Percy](https://percy.io) visual testing for Ruby Selenium.
 
-#### Docs here: [https://docs.percy.io/docs/capybara](https://docs.percy.io/docs/capybara)
+## Installation
+
+npm install `@percy/cli`:
+
+```sh-session
+$ npm install --save-dev @percy/cli
+```
+
+gem install Percy selenium package:
+
+```ssh-session
+$ gem install percy-capybara
+```
+
+## Usage
+
+In your test setup file, require `percy/capybara`. For example if you're using
+rspec, you would add the following to your `spec_helper.rb` file:
+
+
+``` ruby
+require 'percy/capybara'
+```
+
+Now you can use `page.percy_snapshot` to capture snapshots.
+
+> Note: Percy requires JS to be enabled for snapshots to be captured.
```

Comment:
This is true of all SDKs, is there a reason we're specifically pointing it out here?